docs/source/release/v6.12.0/Framework/Algorithms/New_features/38198.rst

```diff
@@ -0,0 +1 @@
+- A new property ``UngroupDetectors`` was added to :ref:`ExtractMask <algm-ExtractMask>` allowing the output ``MaskWorkspace`` to expand the spectra to individal detectors.
```

Comment:
Same kind of thing as previous comments: suggest: "allowing the output ``MaskWorkspace`` to represent the mask flags of the individual detectors."